packages/client/src/generation/generateClient.ts

```diff
@@ -124,12 +123,36 @@ export async function buildClient({
     edge: true,
   })
 
+  const trampolineTsClient = new TSClient({
+    ...nodeClientOptions,
+    reusedTs: 'index',
+    reusedJs: '#main-entry-point',
+  })
+
+  const exportsMapBase = {
+    node: './index.js',
+    'edge-light': './wasm.js',
+    workerd: './wasm.js',
+    worker: './wasm.js',
+    browser: './index-browser.js',
+    default: './index.js',
+  }
+
+  const exportsMapDefault = {
+    require: exportsMapBase,
+    import: exportsMapBase,
+    default: exportsMapBase.default,
+  }
+
   const pkgJson = {
     name: getUniquePackageName(datamodel),
     main: 'index.js',
     types: 'index.d.ts',
     browser: 'index-browser.js',
-    exports: clientPackageExports,
+    exports: {
+      ...clientPkg.exports,
+      ...{ '.': exportsMapDefault },
```

Comment:
This is just for backwards compatibility and must be removed before `driverAdapters` GA.